### ISSUE TITLE
[Void Scans] Connector not working (dead website)

### ISSUE THREAD

**Issue by tryguy** [manga-download/hakuneko#5137]
**Did you read [the troubleshooting guide](https://hakuneko.download/docs/troubleshoot/)**
Yes

**Is the website of the connector working properly / are you able to see the manga within your browser**
Yes

**Describe the bug**
When I select "Void Scans" from the website list, it doesn't bring up the manga available on that site.

I suspect that it's maybe not looking at the right site address for it. It should be looking at "void-scans.com", the it comes back with an error that says "voidscans.com".

**To Reproduce**
1. Name of connector - Void Scans
2. Name of manga (if connector update is working)
3. Name of chapter (if manga update is working)
4. Copy any relevant error message that appears in a popup or (advanced) within the console (press F12)

**Screenshots**
If complex to describe, add screenshots to help explain your problem.

**HakuNeko (please complete the following information):**
 - Version [e.g. 6.17] - 6.1.7

**Comment by MikeZeDev** [comment]
There is "Infernal void scans" connector, which resolve to https://void-scans.com.

Voidscans.com doesnt answer.

**Comment by tryguy** [comment]
Umm, well, I tried "Void Scans" and that produces an error as you say.

But, looking at the networking (f12) when trying to use "Infernal Void Scans", sees to point to: http://ww7.infernalvoidscans.com/

And, what we nees is something to point to void-scans.com.

Actually, I got a notification that it had been corrected, and shut the program down, and restarted, but didn't see any correction. Is the program automatically updated by the changes on github, or am I supposed to do something else (like take the files from this place, and hand-plug the into the install folder), to get any such changes?

**Comment by tryguy** [comment]
Sorry for all the edits. I've just realized that the scan group does go by "Infernal Void Scans" (on patreon) And "Void scans" on their comics.

But "infernalvoidscans.com" is not used anymore. But "void-scans.com" does have their translations. So, probably both "Infernal Void Scans" and "Void Scans" (listed in the manga list should probably point to the current site: "void-scans.com"

And, with that, I'll leave it to you guys.

**Comment by MikeZeDev** [comment]
TLDR : 
* Connector "Infernal void scans" already works , url : void-scans.com.
* Connector "void scans", point to voidscans.com and the website is dead.

We may do some renaming/ deletion. I'll let that to the real devs here. in the meantime use "Infernal void scans".

**Comment by tryguy** [comment]
OK cool. But, am I correct to think that I just exit the program and restart (to let it update) to see this change you've made?

Because I've done that a few times, and for me, when I try to get the manga to appear when using "Infernal Void Scans", I just get network activity trying to access this: http://ww7.infernalvoidscans.com/

It's as if it's still pointing to that (old, no-working site), instead of this change you're saying is already in place? Is there a page I should be reading that will explain how to update to the latest changes?

**Comment by MikeZeDev** [comment]
I changed nothing. Use "Infernal void scans", it points to the good website already.

https://github.com/manga-download/hakuneko/blob/9f21ea3b236651539d4221530255bed024b6094f/src/web/mjs/connectors/InfernalVoidScans.mjs

You may want to use nightly to make sure you have latest changes faster.

**Comment by tryguy** [comment]
Ah cool. I switched over to the nightly build, and now I can access the void scan's stuff. Thanks for the help. I was stuck using the stable, assuming it would update, but yeah, I guess you need the nightly to get the most recent changes.

Thanks again.
